frontend/public/style/_vars.scss

```diff
@@ -18,7 +18,7 @@ $consoleWebDistPath: './';
 $fa-font-path: $consoleWebDistPath + '/';
 // Use PatternFly 4 font family in PatternFly 3.
 $font-family-base: var(--pf-global--FontFamily--sans-serif);
-// Patternfly 3 defaults to 12px, which is too small in many cases.
+// Patternfly 4 defaults to 16px
```

Comment:
This var is specific to PF3, so the comment should be about PF3?  Seems a little confusing that the comment is about a version that isn't impacted by the var.

Reply:
Agreed, intent was to say we default to 14px and while PF4 is 16px. Removing it since `$font-size-base` is PF3